lib/tasks/quality.rake

```diff
@@ -20,13 +20,18 @@ namespace :quality do
     DiscoveryEngine::Quality::SampleQuerySet.new(month:, year:, table_id:).create_and_import
```

Comment:
I'm not always 100% sure how rake passes args in, but doesn't this need to be `table_id: args[:table_id]`?

Reply:
Oh, ignore this - I didn't see this was already there unchanged on line 14.